bazel/repositories.bzl

```diff
@@ -105,8 +105,9 @@ def envoy_api_deps(skip_targets):
     native.git_repository(
         name = "envoy_api",
         remote = REPO_LOCATIONS["envoy_api"],
-        commit = "d4988844024d0bcff4bcd030552eabe3396203fa",
+        commit = "f2c1dc99df64144f76107725bde870dd6827e044",
```

Comment:
Note that this is not the tip of data-plane-api. There is some bazel magic involved and I dont want to get into that now. If someone tells me what to copy paste in the WORKSPACE or this file, I would be happy to sync to master.

Reply:
Can you file a bug and assign to @kyessenov on this? Thanks.

Reply:
It looks like the data-plan-api HEAD is pulling in the Go rules to generate protobuf somethings. Copying the `io_bazel_rules_go` bits from `data-plane-api/WORKSPACE` should let you update this to HEAD if you want.

Reply:
This needs to be added to `api_dependencies()` or `envoy_dependencies()` (as already discussed offline with @rshriram), we're trying to avoid churn in the top-level `WORKSPACE`, as this implies churn in all projects that depend, e.g. to add custom filters.

Reply:
Please try copying lines https://github.com/envoyproxy/data-plane-api/blob/master/WORKSPACE#L39-L48 to the end of WORKSPACE in envoy. If it doesn't work, assign an issue to me.

Reply:
@kyessenov see above comment, we don't want to put this in WORKSPACE.

Reply:
yes and doing that also had some weird bazel errors. I can paste them in the issue to give that person some head start, who ever it will be.